packages/python/plotly/plotly/tests/test_optional/test_px/test_px_input.py

```diff
@@ -138,6 +139,45 @@ def test_name_heuristics():
     assert fig.data[0].hovertemplate == "y=%{marker.size}<br>x=%{y}<extra></extra>"
 
 
+def test_performance_b64():
+    rand_arr_1 = np.array(np.random.random(100000))
+    rand_arr_2 = np.array(np.random.random(100000))
+    b64_arr_1 = b64(rand_arr_1)
```

Comment:
The `rand_arr_1` is already a numpy array, right?
I think we should cast the numpy array to a list using `.tolist()` and then compare the results of numpy(b64) vs normal arrays.

Reply:
You're totally right - no need to convert it to a numpy array!
But as for the casting `.tolist()`, this works for the speed performance test but fails on size performance tests (in those cases it performs the same). I'll leave the speed performance with the `.tolist()` and leave it out for the size performance.